src/translations/main_de.ts

```diff
@@ -458,7 +450,7 @@ Möglicherweise ist OpenSSL nicht installiert.</translation>
     </message>
     <message>
         <source>Updating the database...</source>
-        <translation>Datenbank wird aktualisiert …</translation>
+        <translation>Datenbank wird aktualisiert…</translation>
```

Comment:
Wenn kein Wort verkürzt wird, dann kommt ein Leerzeichen vor den Auslassungspunkten.

Reply:
Wenn es sich bei den Punkten um Auslassungspunkte handeln würde, dann hättest du Recht. Hier wird aber nicht ein Teil vom Wort oder Satz ausgelassen, sondern die Punkte drücken eine laufende Aktion aus. Ich würde sagen, in dem Fall sollte jedem klar sein, dass hier nicht das letzte Wort abgeschnitten wurde.

Reply:
Genau deswegen kommt das Leerzeichen vor die Punkte. Ohne Leerzeichen ist es falsch.

Reply:
Darum geht es ja, dein Einwand wäre richtig wenn es sich bei den drei Punkten um Auslassungspunkte im grammatikalischen Sinne handeln würde, also wenn wir dadurch den Rest vom Satz auslassen würden. Das ist hier aber nicht der Fall. `Datenbank wird aktualisiert` ist der komplette Satz, danach geht es nicht weiter. Folglich handelt es sich hier nicht mehr um Auslassungspunkte, sondern eben um eine Symbolik, die eine fortlaufende Aktion darstellt.
Ich glaube, dass diese Art der Anzeige eher vom Englischen übernommen wurde, wie so vieles im Software-Bereich. Auch bekanntere Programme wie z. B. [apt](https://heise.cloudimg.io/width/1920/q75.png-lossy-75.webp-lossy-75.foil1/_www-heise-de_/imgs/71/2/3/0/9/3/3/2/VirtualBox_Ubuntu_17.10_03_11_2017_11_03_30-b226987c9914d4cc.png) unter Linux fügen kein Leerzeichen vor den Punkten ein.

Reply:
Du hast das schon richtig erkannt. Da der Dreipunkt hier nicht als Auslassung verwendet wird ist es genau der Fall in dem ein Leerzeichen vor den Dreipunkt kommt.

Reply:
https://www.duden.de/sprachwissen/rechtschreibregeln/auslassungspunkte

Reply:
https://www.typolexikon.de/auslassungspunkte/

Reply:
https://www.schreibwerkstatt.co.at/2013/02/04/auslassungspunkte-drei-p%C3%BCnktchen-und-viele-fragezeichen/
Zitat: 3. Vor und nach den Auslassungszeichen gehört ein Leerzeichen.
Nur, wenn ein Wort nicht fortgeführt wird, verzichtet man auf das Leerzeichen.

Reply:
http://www.bibliothek.germanistik.rwth-aachen.de/auslassungszeichen.html

Reply:
Ich glaube du hast meine Antwort nicht richtig durchgelesen. Die drei Punkte sind keine Auslassungszeichen! Um mal aus dem ersten Link zu zitieren:
> Drei Auslassungspunkte zeigen an, dass in einem Wort, Satz oder Text Teile ausgelassen worden sind <§ 99>.

Ein Auslassungszeichen benutzt man also, um Teile eines Satzes wegzulassen, was hier aber nicht der Fall ist (siehe vorherige Antwort).

Hier handelt es sich aber meiner Meinung nach um etwas komplett Anderes. Die drei Punkte drücken eine andauernde, fortlaufende Aktion aus. Die Symbolik wird in Terminal-Programmen verwendet, die sonst keine Möglichkeit haben, einen Fortschritt anzuzeigen und entstammt vermutlich ursprünglich aus dem Englischen, weswegen kein Leerzeichen eingefügt wird.

Reply:
Englische Rechtschreibung ist nicht gleich der deutschen Rechtschreibung! Ich verstehe dich schon. Du beschreibst genau den Fall mit Leerzeichen.

Reply:
Ich glaube, dass die deutsche Rechtschreibung in dem Fall eben nicht greift. Es ist quasi ein Anglizismus.